nifi-extension-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/credentials/factory/CredentialPropertyDescriptors.java

```diff
@@ -91,4 +91,24 @@ private CredentialPropertyDescriptors() { }
             .sensitive(true)
             .build();
 
+    public static final PropertyDescriptor DELEGATION_STRATEGY = new PropertyDescriptor.Builder()
+            .name("delegation-strategy")
+            .displayName("Delegation Strategy")
+            .required(false)
```

Comment:
Since it's a property with allowable values, I believe the convention is to have it as required = true.